rllib/models/torch/recurrent_net.py

```diff
@@ -279,6 +279,9 @@ def get_initial_state(self) -> Union[List[np.ndarray], List[TensorType]]:
         ]
         return h
 
+    def get_initial_state_space(self):
+        return [gym.spaces.Box(-1, 1, shape=(self.cell_size,), dtype=np.float)] * 2
```

Comment:
Hey @smorad , this is great! Thanks for this functionality extension. Some questions:

* Could you add a docstring here, explaining, when to override this method?
* Should we add this to `models/tf/recurrent_net.py` as well?